src/azure-cli/HISTORY.rst

```diff
@@ -74,6 +74,8 @@ Release History
 **Cosmos DB**
 
 * Fix missing --type option for deprecation redirections
```

Comment:
Please remove this one. You can use title or the first comment to add history.

Reply:
Please fix CI as well.

Reply:
Fixed CI, removed specified line from the comment, changed history to use the title.